packages/editor/src/components/save-publish-panels/index.js

```diff
@@ -33,6 +35,9 @@ export default function SavePublishPanels( {
 				select( editorStore ).isPublishSidebarOpened(),
 			hasNonPostEntityChanges:
 				select( editorStore ).hasNonPostEntityChanges(),
+			hasPostMetaChanges: unlock(
+				select( editorStore )
+			).hasPostMetaChanges(),
```

Comment:
I assume we should get `hasPostMetaChanges` from here and not in the component props, right?

Reply:
I just changed it. Feel free to revert it if you consider it necessary.